src/test/librados_test_stub/TestRadosClient.cc

```diff
@@ -292,7 +292,7 @@ void TestRadosClient::flush_aio_operations(AioCompletionImpl *c) {
 int TestRadosClient::aio_watch_flush(AioCompletionImpl *c) {
   c->get();
   Context *ctx = new LambdaContext(std::bind(
-    &TestRadosClient::finish_aio_completion, this, c, _1));
+    &TestRadosClient::finish_aio_completion, this, c, boost::placeholder::_1));
```

Comment:
this change is wrong, and not necessary.

i added
```c++
using namespace std::placeholders;
```
in 5c1a5bcf18b79e9deb70143f40ae01bbad74444b